nix/pkgs/haskell/haskell.nix

```diff
@@ -59,9 +58,138 @@ let
       "https://github.com/input-output-hk/Win32-network"."3825d3abf75f83f406c1f7161883c438dac7277d" = "19wahfv726fa3mqajpqdqhnl9ica3xmf68i254q45iyjcpj1psqx";
       "https://github.com/input-output-hk/hedgehog-extras"."edf6945007177a638fbeb8802397f3a6f4e47c14" = "0wc7qzkc7j4ns2rz562h6qrx2f8xyq7yjcb7zidnj7f6j0pcd0i9";
     };
+    # Configuration settings needed for cabal configure to work when cross compiling
+    # to windows. We can't use `modules` for these as `modules` are only applied
+    # after cabal has been configured.
+    cabalProjectLocal = lib.optionalString pkgs.stdenv.hostPlatform.isWindows ''
+      -- When cross compiling we don't have a `ghc` package
```

Comment:
the conditional is "building for windows" not "cross-building" - which one should it be? Presumably if you were doing a native build on windows this would be okay?